requirements-dev.txt

```diff
@@ -13,3 +13,4 @@ python-dateutil==2.8.2
 ruff==0.0.265
 tox==4.5.1; python_version >= "3.8"
 tox==3.28.0; python_version < "3.8"  # tox 4+ has virtualenv requirements that are incompatible with other pkgs
+linkml-runtime==1.5.0
```

Comment:
```suggestion
```
Since linkml-runtime is installed from `requirements.txt`, it is not needed here. This is just for any libraries that need to be installed for tests and checks to run.